ccm_web/server/src/api/api.create.section.handler.ts

```diff
@@ -27,7 +27,7 @@ export class CreateSectionApiHandler {
       logger.debug(`Sending request to Canvas - Endpoint: ${endpoint}; Method: ${method}; Body: ${JSON.stringify(requestBody)}`)
       const response = await this.requestor.requestUrl<CanvasCourseSection>(endpoint, method, requestBody)
       const { id, name } = response.body
-      return { id, name }
+      return { id, name, total_students: 0 }
```

Comment:
I believe this is from a different issue but as usual I did some sort of sloppy merge.
Same with the below.